flink-runtime/src/main/java/org/apache/flink/runtime/state/heap/HeapMapState.java

```diff
@@ -140,25 +141,25 @@ public boolean contains(UK userKey) {
 	@Override
 	public Iterable<Map.Entry<UK, UV>> entries() {
 		Map<UK, UV> userMap = stateTable.get(currentNamespace);
-		return userMap == null ? null : userMap.entrySet();
+		return userMap == null ? Collections.emptyList() : userMap.entrySet();
```

Comment:
Do we need to return `Collections.emptySet` here to align with the `userMap.entrySet()`?